### PR TITLE
ci: fix build error and account for APK as default

### PR DESCRIPTION
**Pull request by dannil**
See separate commits for details. The first commit should fix all the recent build errors for the task `Test x86_64` ([1](https://github.com/openwrt/luci/actions/runs/11852952401/job/33032096379?pr=7394), [2](https://github.com/openwrt/luci/actions/runs/11764799075/job/32770355810), [3](https://github.com/openwrt/luci/actions/runs/11795975811/job/32856854484)). For reviewers, make sure to check that the workflow run is green and produces the expected results.

* ci: upgrade openwrt/gh-action-sdk workflow to v7

    With the change of the Docker container not including pre-built
    binaries [1], the previous version of the workflow fails as it still
    expects those to be present in the upstream container while the new
    versions runs a file called setup.sh if present to configure the
    container.

    By upgrading openwrt/gh-action-sdk workflow to v7, we make sure that
    setup.sh is run and configures the container [2].

    [1] https://github.com/openwrt/docker/commit/9b55784b18f8d2c684aac4dd21a224320bb9b9ce
    [2] https://github.com/openwrt/gh-action-sdk/commit/0c00b28c11bed5c7697e807d4cb8ef176ca688ef

* ci: account for APK as default on main branch

    With APK defaulted on openwrt:main, we should include packages with the
    .apk file extension as artifacts. Also default the logging to V=s as
    otherwise the APK version check won't be printed to stdout.

- [X] This PR is not from my *main* or *master* branch :poop:, but a *separate* branch :white_check_mark:
- [X] Each commit has a valid :black_nib: `Signed-off-by: <my@email.address>` row (via `git commit --signoff`)
- [X] Each commit and PR title has a valid :memo: `<package name>: title` first line subject for packages
- [ ] Incremented :up: any `PKG_VERSION` in the Makefile N/A
- [ ] Tested on: (architecture, openwrt version, browser) :white_check_mark: N/A
- [ ] \( Preferred ) Mention: @ the original code author for feedback
- [ ] \( Preferred ) Screenshot or mp4 of changes:
- [ ] \( Optional ) Closes: e.g. openwrt/luci#issue-number
- [ ] \( Optional ) Depends on: e.g. openwrt/packages#pr-number in sister repo
- [X] Description: (describe the changes proposed in this PR)

### CODE REVIEW

**Comment by dannil** [comment]
For reference, here's how the APK version check looks like today, exemplified by manipulating `PKG_VERSION` for `luci-app-adblock-fast` to an invalid value.

* today: https://github.com/dannil/luci/commit/6fd7271edb15403b77c68b2fb7fe7ebe0d1938c6: https://github.com/dannil/luci/actions/runs/11864787802/job/33068858146#step:5:8635
* with `V=s`: https://github.com/dannil/luci/commit/5b6f1d2231a16e1222df5704bff5e3d71c39d5d0: https://github.com/dannil/luci/actions/runs/11864801201/job/33068895111#step:5:16424

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -54,15 +54,18 @@ jobs:
         env:
           ARCH: ${{ matrix.arch }}-${{ env.BRANCH }}
           FEEDNAME: packages_ci
+          V: s
 
       - name: Move created packages to project dir
-        run: cp bin/packages/${{ matrix.arch }}/packages_ci/*.ipk . || true
+        run: cp bin/packages/${{ matrix.arch }}/packages_ci/* . || true
```
Comment (hnyman) [inline .github/workflows/build.yml:60]: I wonder if this should be `*.?pk` instead ?  
To match both `.apk` and `.ipk`, but not removing the selection criteria totally leaving just the `*` wildcard there.

Comment (Ansuel) [reply .github/workflows/build.yml:60]: Should we bite the bulled and be more explicit?
```
cp bin/packages/${{ matrix.arch }}/packages_ci/*.ipk . || true
cp bin/packages/${{ matrix.arch }}/packages_ci/*.apk . || true
```

Comment (systemcrash) [reply .github/workflows/build.yml:60]: Prefer one-liners :D

Comment (dannil) [reply .github/workflows/build.yml:60]: I'm fine with either solution, I don't have any real preference as long as we can capture the .apk files.

Comment (dannil) [reply .github/workflows/build.yml:60]: I saw that actions-shared-workflows uses the [same principle](https://github.com/openwrt/actions-shared-workflows/blob/main/.github/workflows/multi-arch-test-build.yml#L119) and then [copies the relevant files from the working directory](https://github.com/openwrt/actions-shared-workflows/blob/main/.github/workflows/multi-arch-test-build.yml#L165-L171) so maybe we should keep it the way it is now?

Comment (Ansuel) [reply .github/workflows/build.yml:60]: Well if we want to keep consistency yes, we should use the * directly.

**Comment by systemcrash** [comment]
Awesome

**Pull request merged**